advocacy_docs/partner_docs/FlywayGuide/04-Configuration.mdx

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Scott - Reworked this page.  
Made configuration steps
moved description of options 
added some wording to clarify.